src/renderer/extensions/vueNodes/widgets/components/WidgetInputNumberInput.vue

```
@@ -3,6 +3,7 @@ import { onClickOutside } from '@vueuse/core'
 import { computed, ref, useTemplateRef } from 'vue'
 import { useI18n } from 'vue-i18n'
 
+import Button from '@/components/ui/button/Button.vue'
```

ESLint

[error] Unable to resolve path to module '@/components/ui/button/Button.vue'.

(import-x/no-unresolved)